src/lib/viewers/ActionSummaryViewer.ts

```diff
@@ -75,6 +75,64 @@ export class RulesActionSummaryViewer extends AbstractActionSummaryViewer {
 			this.display.displayLog(indent(getMessage(BundleName.ActionSummaryViewer, 'rules-action.rules-item', [ruleCountForEngine, engineName])));
 		}
 	}
+}
+
+export class RunActionSummaryViewer extends AbstractActionSummaryViewer {
```

Comment:
This code is _almost_ identical to the code from `RunSummaryViewer.ts`, but it's different to accomplish the goals of the story.